pkg/sloop/processing/eventcount_test.go

```diff
@@ -116,16 +107,93 @@ func Test_EventCountTable_Event(t *testing.T) {
 
 }
 
+func Test_EventCountTable_TruncateEvent(t *testing.T) {
```

Comment:
This test is hard to read, as the timestamp names don't tell me much (firstTs, lastTs).  And this feels like a few different tests put into one.

Reply:
I might split this into 4 tests:

1. No data, first event creates 1 partition
1. No data, first event creates 2 partitions
1. There is one partition already in the store
1. There are two partitions in the store

This might be easier with hour partitions, and timestamp variable names that clearly tell me what they are (maybe timestamp_h03m02 or something)